cpp/src/neighbors/detail/ann_utils.cuh

```diff
@@ -148,7 +148,7 @@ struct config<float> {
 };
 template <>
 struct config<half> {
-  using value_t                    = half;
+  using value_t                    = float;
```

Comment:
Please check whether this is used outside the IVF-Flat. Changing the accumulation type like this can have a drastic impact on performance.

Reply:
I did simple benchmark and it showed no significant differences. I'll use cuvs ann-bench to get a more detailed benchmark results later

Reply:
@cjnolet hi, push it to 25.10, thx